lib/workers/pr/changelog/types.ts

```diff
@@ -3,6 +3,8 @@ export interface ChangeLogNotes {
   id?: number;
   name?: string;
   tag?: string;
+  // url to changelog.md file or github/gitlab release api
+  notesSourceUrl: string;
```

Comment:
```suggestion
  notesApiUrl: string;
```

I think this is a better name, as you point to the releases api endpoint. Maybe this shouild only point to the repo?

Reply:
notesSourceUrl can be link to release gitlab/github api, or link to changelog.md file (for example `https://github.com/facebook/react/blob/main/CHANGELOG.md`)

This should not point to the repo, because in one repo can be several packages (monorepo) with their own chanelog.md files

Current logic for deduplicate release notes compare only repo source url and such bugs https://github.com/renovatebot/renovate/issues/10954  appears